Makefile

```diff
@@ -25,13 +25,15 @@ install-with-mruby:
 	go install -tags mrb ./...
 
 build:
-	env go build -tags mrb -ldflags $(LD_FLAGS) -o $(OUTPUT) cmd/anycable-go/main.go
+	go build -tags mrb -ldflags $(LD_FLAGS) -o $(OUTPUT) cmd/anycable-go/main.go
 
 prepare-cross-mruby:
```

Comment:
So, let's get rid of this too.

And here too: https://github.com/anycable/anycable-go/blob/master/etc/build_config.rb